src/uct/rocm/base/rocm_base.c

```diff
@@ -227,6 +227,11 @@ ucs_status_t uct_rocm_base_mem_query(uct_md_h md, const void *addr,
         mem_attr_p->alloc_length = length;
     }
 
+    if (mem_attr_p->field_mask & UCT_MD_MEM_ATTR_FIELD_DMABUF_FD) {
+        /* unsupported */
```

Comment:
I am assuming that setting UCT_MD_MEM_ATTR_FIELD_DMABUF_FD without UCT_MD_MEM_ATTR_FIELD_DMABUF_OFFSET and vice versa is illegal, isn't it?

Reply:
Let's explicitly document this.